.babelrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This can be removed once a new release of Jest is available. Let's not merge this PR until then.